src/main/java/xyz/nucleoid/codecs/MoreCodecs.java

```diff
@@ -91,6 +93,15 @@ public final class MoreCodecs {
         }
     });
 
+    public static final Codec<Box> BOX = RecordCodecBuilder.create(instance -> instance.group(
+            Codec.DOUBLE.fieldOf("minX").forGetter(box -> box.minX),
+            Codec.DOUBLE.fieldOf("minY").forGetter(box -> box.minY),
+            Codec.DOUBLE.fieldOf("minZ").forGetter(box -> box.minZ),
+            Codec.DOUBLE.fieldOf("maxX").forGetter(box -> box.maxX),
+            Codec.DOUBLE.fieldOf("maxY").forGetter(box -> box.maxY),
+            Codec.DOUBLE.fieldOf("maxZ").forGetter(box -> box.maxZ)
```

Comment:
Nit: what about something like this? 🙂 Gives the `"min": [0, 0, 0]` syntax, which is maybe a bit easier to read 
```suggestion
            Vec3d.CODEC.fieldOf("min").forGetter(box -> new Vec3d(box.minX, box.minY, box.minZ)),
            Vec3d.CODEC.fieldOf("max").forGetter(box -> new Vec3d(box.maxX, box.maxY, box.maxZ))
```

Reply:
Sure, I agree that it's nicer - I've rebased this onto 1.19.3 and applied this change :)